index.html

```diff
@@ -251,5 +251,4 @@ <h1 class="header-4">I'm always interested in hearing<br>about new projects, so
     </section>
     </div>
 </body>
-
-</html>
+</html>
```

Comment:
- Please allow the footer to span 100% width in the desktop mode, as at percent it is squashed in as shown in the image below. 

- Also please add the gray line on top of the social media icon section on the footer as it is on the Figma.You should try as much to make your design look like that of the Figma design, this will help you attend to the details of a design in the future no matter how little. 

**Your Design** 



![image](https://github.com/NgachaIan/PersonalPortfolio/assets/58771507/d2083a23-17e4-453f-aef1-3044df3d4ae6)



**Figma Design**
![image](https://github.com/NgachaIan/PersonalPortfolio/assets/58771507/c4d1b71c-6df6-4258-85ad-c82ff6fa1537)